spk/fuse-exfat/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = fuse-exfat
+SPK_VERS = 1.2.1
+SPK_REV = 2  
+SPK_ICON = src/fuse.png
+
+DEPENDS = cross/$(SPK_NAME) cross/exfat-utils
+
+MAINTAINER = Roman Tymoshyk 
+DESCRIPTION = Add support of exFAT filesystem
```

Comment:
This is a description of the package as it appears on [our repository](https://synocommunity.com/packages), or in Package Center. You might want to change it to the description the developer provides.